source/content/guides/composer-convert.md

```diff
@@ -169,19 +214,31 @@ If you are using an exported config, you will need to move the configuration fil
 
 Locate the configuration files in your existing site and move them here. If they are stored in the files directory on your existing site, retrieve them via [SFTP](/sftp), as the Git clone would not contain them. The example project is configured to use this location.
 
-## Update to Latest Drupal Core
+## Set up a Multidev (Optional)
 
-The Pantheon Drupal 8 Composer repository may not depend on the latest secure release of Drupal Core. To keep your site safe, it's a good idea to pull the latest version in now -- before moving on -- by running:
+If your Pantheon account has access to [Multidev](/multidev), create a Multidev to push your new code to:
 
 ```bash{promptUser:user}
-composer update drupal/core --with-dependencies
+git push origin composerify && terminus env:create $site.dev composerify
 ```
 
-## Prepare to Deploy
+This will set up the Multidev environment to receive and demo our changed code.
+## Commit
+
+Commit your work to the Git repo. From the `$site` directory, run the following:
+
+```bash{promptUser: user}
+cp -r .git ../$site-composer/
+cd ../$site-composer
+git add .
+git commit -m "Convert to Composer based install"
+```
 
-At this point, your new project directory should contain all of the unique code from your existing Drupal 8 site, plus all of the code required to make a Composer driven project work. Since Pantheon requires all runtime code to be present when deploying to the platform, if no CI solution is a part of your workflow, you must now modify the project to be deployable straight to Pantheon.
+You should see a large number of files committed to the new branch created earlier.
 
-If you do plan on using a CI solution, refer to our [Build Tools](/guides/build-tools) guide at this point.
+## Prepare to Deploy
```

Comment:
The "Prepare to Deploy" step should no longer be necessary. There is no `prepare-for-pantheon` any longer, and `composer install` should have already ran implicitly as one of the previous steps.